### DISCUSSION TITLE
Bell icon request for the Twilio Alarms

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1864] [Request: Icon] [answered]
Hi team, 
We need a bell icon for the Twilio Alarms. The icon will replace the Console left navigation. 
<img width="518" alt="bell" src="https://user-images.githubusercontent.com/67388401/133170773-fe9a0187-17e4-4e62-9df5-108248ff614d.png">

We would like to use https://app.streamlinehq.com/icons/streamline-light/interface-essential/alert/alarm-bell

Thanks!

**Comment by nkrantz** [comment]
We've added this icon to our next sprint and we'll let you know when it's been added to the system!

**Reply by andioneto** [reply]
@sang-yeo In the spirit of ruthless prioritization, we had to push this icon addition to our next sprint, which starts next Wednesday. Thanks for your patience and for helping us make Paste better. We will let you know as soon as the icon is available. 🙇‍♀️

**Reply by sang-yeo** [reply]
Thank you for the update!

**Comment by loreina** [comment, marked as answer]
Hey @sang-yeo, thanks for your patience. The Alarms icon is now live in Figma as `logo/product/Alarms` and in code as `ProductAlarmsIcon`. ⏰ 🥳
